packages/components/src/Dialog/Layout/DialogFooter.tsx

```diff
@@ -24,38 +24,12 @@
 
  */
 
-import React, { FC, ReactNode } from 'react'
 import styled from 'styled-components'
-import { CompatibleHTMLProps } from '@looker/design-tokens'
-import { Space, SpaceHelperProps } from '../../Layout/Space'
-
-export interface DialogFooterProps
-  extends CompatibleHTMLProps<HTMLDivElement>,
-    SpaceHelperProps {
-  /**
-   * Secondary content to place in the footer
-   */
-  secondary?: ReactNode
-}
-
-const DialogFooterLayout: FC<DialogFooterProps> = ({
-  children,
-  secondary,
-  ...props
-}) => {
-  return (
-    <Space as="footer" reverse between {...props}>
-      <Space reverse>{children}</Space>
-      {secondary && <Space>{secondary}</Space>}
-    </Space>
-  )
-}
-
-export const DialogFooter = styled(DialogFooterLayout).attrs(
+import { ModalFooter, ModalFooterProps } from '../../Modal/ModalFooter'
+
+export const DialogFooter = styled(ModalFooter).attrs(
```

Comment:
Although it looks a little silly I generally like to re-export the interface for a component like this with a matching name so that if a downstream consumer wants to use it (and we want to extend it specifically for this component) things stay unified.

```suggestion

// eslint-disable-next-line @typescript-eslint/no-empty-interface
export interface DialogFooterProps extends ModalFooterProps {}
```